exporter/cat1_view_helper.go

```diff
@@ -15,29 +15,20 @@ import (
 	"github.com/projectcypress/cdatools/models"
 )
 
-func escape(i interface{}) string {
-	switch str := i.(type) {
-	case string:
-		return escapeString(str)
-	case int64:
-		return escapeString(strconv.FormatInt(str, 10))
-	case int:
-		return escapeString(strconv.Itoa(str))
-	case *int64:
-		if str != nil {
-			return escapeString(strconv.FormatInt(*str, 10))
-		}
-	case *int:
-		if str != nil {
-			return escapeString(strconv.Itoa(*str))
-		}
-	}
-	return ""
+// int64PtrToString converts the value of an *int64 to a string.
+// NOTE: This function is probably not even needed, since I believe
+// text/template will access the value *int64 points to and not the
+// actual address.
```

Comment:
I seem to recall reading that when I was studying text/template, just didn't know this was in the codebase until now. When I take the function out, tests pass. I believe I also have concrete proof in my other PR (entry-refactor) because allergy has a StartTime and I just access it in the template and it works. I'll go ahead and drop it.